src/devsite-paths.json

```
@@ -75,5 +75,12 @@
         "owner": "AdobeDocs",
         "root": "/src/pages/",
         "branch": "eds-migration"
+    },
+    {
+        "pathPrefix": "/app-builder-template-registry/",
```

```suggestion
        "pathPrefix": "/app-builder-template-registry",
```

Convention in this file is to remove the trailing slashes from the path prefixes